### PR TITLE
[api-v1-collection] add new fullTextSearch

### PR DESCRIPTION
**Pull request by iskurenkov**
1. Changed searchCollections path, removed /search from path 
2. Changed full test search in collections

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -505,6 +505,31 @@ paths:
         '500':
           $ref: "#/components/responses/ServerError"
 
+  /v1/collections/:
```
Comment (evgenynacu) [inline openapi.yaml:508]: it should not end with slash

**Review by evgenynacu — CHANGES_REQUESTED** [review]

File: openapi.yaml (hunk 1 above)
Comment (iskurenkov) [reply openapi.yaml]: Fixed.

File: openapi.yaml [hunk 2]
```diff
@@ -2484,6 +2511,53 @@ components:
...
+          items:
+            $ref: "#/components/schemas/Address"
+        minter:
+          default: []
+          description: "Allows you to find private collections where specific minter can mint tokens"
+          type: array
+          items:
+            $ref: "#/components/schemas/Address"
+        text:
+          default: null
```
[the remark this answers is not in the document]
Comment (iskurenkov) [reply openapi.yaml]: Judging by the specification, the "default" parameter for the Schema Object is optional. Removed.

**Review by evgenynacu — CHANGES_REQUESTED** [review]

File: openapi.yaml [hunk 3]
```diff
@@ -467,7 +467,7 @@ paths:
         '500':
           $ref: "#/components/responses/ServerError"
 
-  /v0.1/collections/sync:
+  /v1/collections:
```
[the remark this answers is not in the document]
Comment (iskurenkov) [reply openapi.yaml:470]: Испправил

File: openapi.yaml [hunk 4]
```diff
@@ -2484,6 +2511,41 @@ components:
...
           "$ref": "#/components/schemas/Address"
 
+    CollectionSearchForm:
+      type: object
+      properties:
+        cursor:
+          default: null
+          description: "Combined continuation token to paginate activities search result"
+          type: string
+        blockchain:
```
Comment (evgenynacu) [inline openapi.yaml:2521]: blockchains

File: openapi.yaml [hunk 5]
```diff
@@ -2484,6 +2511,41 @@ components:
...
+          default: []
+          description: "Allows you to limit the search to selected blockchains"
+          type: array
+          items:
+            $ref: "#/components/schemas/Blockchain"
+        type:
```
Comment (evgenynacu) [inline openapi.yaml:2527]: types

File: openapi.yaml [hunk 6]
```diff
@@ -2484,6 +2511,41 @@ components:
...
+          default: []
+          description: "Allows you to find collections by type"
+          type: array
+          items:
+            $ref: "#/components/schemas/CollectionType"
+        owner:
```
Comment (evgenynacu) [inline openapi.yaml:2533]: owners

**Review by evgenynacu — CHANGES_REQUESTED** [review]

File: openapi.yaml (hunk 4 above)
Comment (iskurenkov) [reply openapi.yaml:2521]: Fixed

File: openapi.yaml (hunk 5 above)
Comment (iskurenkov) [reply openapi.yaml:2527]: Fixed

File: openapi.yaml (hunk 6 above)
Comment (iskurenkov) [reply openapi.yaml:2533]: Fixed

**Pull request merged**